src/app/(market-store)/shop/page.tsx

```
@@ -0,0 +1,17 @@
+import type { Metadata } from "next";
+import Link from "next/link";
+
+type Props = {};
+
+export const metadata: Metadata = {
+  title: "Shop",
+};
+export default function ShoppingPage({}: Props) {
```

Biome (2.1.2)

[error] Unexpected empty object pattern.



(lint/correctness/noEmptyPattern)